main.js

```diff
@@ -1,82 +1,157 @@
 // use the variables below to run Math function
-let val1;
-let op;
+let val1 = '';
```

Comment:
I find myself a little confused while reading the code about what each of these variables are used for.

Let's definitely do a screen share and you can walk me through your logic, and we can talk about maybe changing up the variable names and adding some comments as to what each one does